dakota/oat/cooling-time/ct-df-pu-sa.csv

```diff
@@ -0,0 +1,6 @@
+CT,Max Pu in all CP,Pu Quality in all CP at Max Pu,Max Pu in HLW,Pu Quality in HLW at Max Pu,Max Pu in all RPR,Pu Quality in all RPR at Max Pu
```

Comment:
Are these dakota input data files? Do you have any editorial control over what the column names are? I find it problematic that they have spaces and abbreviations. What's a CP? Cooling pool?
Also, these need units. (mass in kg? tonnes?)